internal/entities/properties/properties.go

```diff
@@ -210,6 +210,15 @@ type newPropertiesConfig struct {
 	skipPrefixCheck bool
 }
 
+// MustNewProperties creates Properties from a map or panics
+func MustNewProperties(props map[string]any, opts ...newPropertiesOption) *Properties {
```

Comment:
not against adding this to the properties module, but would it suffice to have the method in the tests?

Reply:
I'm not able to import from a `_test.go` file. I moved this where it's used.